src/plugins/rovpilot-wire/index.js

```diff
@@ -44,7 +44,7 @@
     deps.globalEventLoop.on('mcu.status', function (status) {
       if ('targetDepth' in status) {
         self.state.depth.enabled = status.targetDepth != DISABLED;
-        self.state.depth.targetDepth = Number(status.targetDepth) / 100;
+        self.state.depth.targetDepth = Number(status.targetDepth);
```

Comment:
When we first wrote depth hold that was simply an easy way to pass the information.  At that time we still had no pattern for passing floats back and forth.